lightwood/mixers/helpers/default_net.py

```diff
@@ -58,6 +59,8 @@ def __init__(self, dynamic_parameters,
 
             layers = []
             for ind in range(len(shape) - 1):
+                if (dropout is not None) and (not deterministic) and (0 < ind < len(shape)):
```

Comment:
Ok... two questions:

1. Why not apply dropout when `deterministic==True` ?

2. Why was `deterministic` set to `False` as the default ? (might have been me, granted, but if so it was a mistake)

We should aim for lightwood's results to be replicable.
Granted, `deterministic` might not be needed at all, since we set torch to be deterministic in `__init__.py`

So I guess we should solve `1.` and then just remove `deterministic` and move all seeding logic into `__init__.py`, since most is already replicated there (no pun intended).

Reply:
Also this should check `if dropout > 0` and set the default value to `0`, probably make more sense. but maybe not ? It just seems "better" to me that way style/logic wise, but if you disagree @paxcema then leave it as is.